app/assets/stylesheets/components/curriculum_details.scss

```diff
@@ -51,10 +50,13 @@
 
       h3 {
         margin-bottom: .2em;
-      }
-
-      p {
-        font-size: .9em;
+        margin-top: 20px;
+        font-size: 18px;
+        font-weight: bold;
+        letter-spacing: 0.2px;
+        font-family: OpenSans;
+        color: #4a4a4a;
+        color: var(--greyish-brown);
```

Comment:
In the design on Zeplin, it mentioned - 

color: #4a4a4a;
color: var(--greyish-brown);

in the CSS code. 
I was wondering if both are necessary or not.